packages/core/demo/demo-data/meter.ts

```diff
@@ -0,0 +1,37 @@
+import { colors } from "./colors";
+
+export const meterData = {
+	labels: ["CPU"],
+	datasets: [
+		{
+			label: "CPU",
+			fillColors: [colors[0]],
+			data: {
+				min: 0,
+				max: 1500,
+				peak: 1200,
+				value: 550,
+				status: [
+					{ range: [0, 1000], status: "success" },
+					// { range: [500, 1000], status: "custom" },
+					{ range: [1000, 1200], status: "warning"},
+					{ range: [1200, 1500], status: "danger"}
+				],
+			}
+		}
+	]
+};
```

Comment:
If we're rendering only 1 meter when using the `MeterChart` class then why do we have the concept of datasets in here?

Reply:
too try and keep the data layer as uniform as possible. The same way pie/donut use datasets but only render the first dataset as a visualisation. Also I kept fillColors using an array of colors even though meter will only use the first color.